### PR TITLE
[Port] Eoran stuff from Stonekeep | Acolytes, Templars, Clerics

### PR DESCRIPTION
**Pull request by Vladmar11**
Adds Eora Acolytes/Templars/Clerics to the game and associated spells + sprites.

Changed Eora's description. Now Eora is the goddess of love, life and beauty as everything is misleading that Eora and Baotha are one and the same.

Spells:

Eoran Bloom:

With a long cooldown and casting time, summon a Eoran Bud to help promote "peace" and pacify people. 

Eoras Curse:

Makes the target high and drunk. That's all, yeah.

Robes have alternative version (rightclick on it)

(Credit to SELFHELL for code and monochrome9090 for the sprites)

Feat. SayonaraEcho

Plans: 
1) Remove ability to marry people for druids and give this ability to eoran acolytes (from here https://github.com/Darkrp-community/OpenKeep/pull/1038/)
2) Add new sprites for templar's tabards 
3) Review the acolytes/cleric's abilities later (they are too weak)

4e
im6age

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/church/templar.dm [hunk 1]
```diff
@@ -73,6 +73,14 @@
 			head = /obj/item/clothing/head/roguetown/helmet/heavy/templar/ravox
 			H.mind.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
 			beltr = /obj/item/rogueweapon/sword/long
+		if("Eora") //Eora content from stonekeep
```
Comment (Gyrandola) [inline code/modules/jobs/job_types/roguetown/church/templar.dm:76]: Useless comment

File: code/modules/clothing/rogueclothes/hats/hats.dm [hunk 2]
```diff
@@ -888,6 +888,60 @@
...
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	REMOVE_TRAIT(user, TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/proc/peace_check(mob/living/user)
+	// return true if we should be unequippable, return false if not
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, "<span class='warning'>I feel at peace. <b style='color:pink'>Why would I want anything else?</b></span>")
```
Comment (Gyrandola) [inline code/modules/clothing/rogueclothes/hats/hats.dm:933]: Spannify this

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	chargedloop = /datum/looping_sound/invokegen
+	associated_skill = /datum/skill/magic/holy
+	charge_max = 60 SECONDS
+
+/obj/effect/proc_holder/spell/invoked/bud/cast(list/targets, mob/living/user)
+	var/turf/T = get_turf(targets[1])
+	if(!isclosedturf(T))
+		new /obj/item/clothing/head/peaceflower(T)
+		return TRUE
+	to_chat(user, "<span class='warning'>The targeted location is blocked. The flowers of Eora refuse to grow.</span>")
```
Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:24]: Span

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	antimagic_allowed = TRUE
+	charge_max = 10 SECONDS
+	miracle = FALSE
+
+/obj/effect/proc_holder/spell/invoked/eoracurse/cast(list/targets, mob/living/user)
+	if(isliving(targets[1]))
+		var/mob/living/carbon/target = targets[1]
+		target.apply_status_effect(/datum/status_effect/buff/druqks)
+		target.apply_status_effect(/datum/status_effect/buff/drunk)
+		target.visible_message("<span class='info'>A purple haze shrouds [target]!</span>", "<span class='notice'>I feel much calmer.</span>")
```
Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:48]: Span

**Review by Gyrandola — CHANGES_REQUESTED** [review]

**Comment by Gyrandola** [comment]
By spannifying I meant using the span_warning helper instead of raw html

**Comment by Vladmar11** [comment]
> By spannifying I meant using the span_warning helper instead of raw html

Got it. Sorry, this is my first job. Is that right?

**Comment by Coolgat3** [comment]
What kind of drug? If it's spice, then I'll have you know it has a low chance to crash someone for the entiriety of the round. It also gives -5 fortune, which in combat is an extreme hinderance. Basically an eoran acolyte will be able to give someone -5 fortune ad slit their throat with a thorn or something.

**Comment by ophaq** [comment]
Please add a spell that is a half second stun where the person jizzes their pants.

**Comment by Gyrandola** [comment]
fix conflicts and i will test merge this

**Comment by Vladmar11** [comment]
> fix conflicts and i will test merge this

Ready to merge

**Comment by Dilligentdog** [comment]
should give Eoran acolytes the same skill bath wenches get. Fabled lover or whatever the name is. They are followers of the goddess of love afterall

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	antimagic_allowed = TRUE
+	charge_max = 10 SECONDS
+	miracle = FALSE
+
+/obj/effect/proc_holder/spell/invoked/eoracurse/cast(list/targets, mob/living/user)
+	if(isliving(targets[1]))
+		var/mob/living/carbon/target = targets[1]
+		target.apply_status_effect(/datum/status_effect/buff/druqks)
+		target.apply_status_effect(/datum/status_effect/buff/drunk)
+		target.visible_message(span_warning("<span class='info'>A purple haze shrouds [target]!</span>"), span_warning("<span class='notice'>I feel much calmer.</span>"))
```
Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:48]: if you use a span helper, you need to remove the raw html one.

Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:48]:
```suggestion
		target.visible_message(span_info("A purple haze shrouds [target]!"), span_info("I feel much calmer."))
```

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+	associated_skill = /datum/skill/magic/holy
+	charge_max = 60 SECONDS
+
+/obj/effect/proc_holder/spell/invoked/bud/cast(list/targets, mob/living/user)
+	var/turf/T = get_turf(targets[1])
+	if(!isclosedturf(T))
+		new /obj/item/clothing/head/peaceflower(T)
+		return TRUE
+	to_chat(user, span_warning("<span class='warning'>The targeted location is blocked. The flowers of Eora refuse to grow.</span>"))
+	return FALSE
```
Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:25]:
```suggestion
	to_chat(user, span_warning("The targeted location is blocked. The flowers of Eora refuse to grow."))
	return FALSE
```

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 7]
```diff
@@ -0,0 +1,51 @@
+//Eora content from Stonekeep
```
Comment (Gyrandola) [inline code/modules/spells/roguetown/acolyte/eora.dm:1]:
```suggestion
```

File: code/modules/clothing/rogueclothes/hats/hats.dm [hunk 8]
```diff
@@ -889,6 +889,60 @@
...
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	REMOVE_TRAIT(user, TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/proc/peace_check(mob/living/user)
+	// return true if we should be unequippable, return false if not
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, span_warning("<span class='warning'>I feel at peace. <b style='color:pink'>Why would I want anything else?</b></span>"))
```
Comment (Gyrandola) [inline code/modules/clothing/rogueclothes/hats/hats.dm:934]:
```suggestion
			to_chat(user, span_warning("I feel at peace. <b style='color:pink'>Why would I want anything else?</b>"))
```

**Review by Gyrandola — APPROVED** [review]

**Comment by Frienderanian** [comment]
Can Eora still be the Goddess of Lust? I kind of like that. I feel like it can be spun in a positive direction. Lust has driven men to do great things throughout all of history and I kind of like Eora as a god bringing validation to the real feelings of Lust that Men typically have and shouldn't be ashamed of. :)

**Pull request merged**